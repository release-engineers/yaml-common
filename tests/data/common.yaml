example-exists:
  str-1: This property already exists in common.yaml
  str-2: This property exists in all files
