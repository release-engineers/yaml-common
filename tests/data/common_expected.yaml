example-exists:
  str-1: This property already exists in common.yaml
  str-2: This property exists in all files
example-common:
  # common comment
  depth:
    more-depth:
      array:
      - yes
      - for sure
      str: common string
    int: 123
    float: 1.23
  bool: true
  null:
example-mixed:
  array-common:
  - yes
  - for sure again
  str-common: common string
  sre-common-multiline: |
    -----BEGIN PUBLIC KEY-----
    01234567890
    -----END PUBLIC KEY-----
